repos/spack_repo/builtin/packages/py_vhacdx/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyVhacdx(PythonPackage):
+    """Generates an approximate convex decomposition of a triangle mesh."""
+
+    homepage = "https://github.com/trimesh/vhacdx"
+    pypi = "vhacdx/vhacdx-0.0.8.post2.tar.gz"
+
+    maintainers("moloney")
+
+    license("MIT", checked_by="moloney")
+
+    version("0.0.8.post2", sha256="aa27c5ef19ed4aba428fa9408dccc37f2b7a6bbfaddc48b06a8cea9faaf93156")
+
+    depends_on("cxx", type="build")
+
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-pybind11", type="build")
```

Comment:
These dependencies should match the project's `pyproject.toml`'s:

```
[build-system]
requires = [
    "setuptools>=42",
    "wheel",
    "pybind11>=2.13.1",
]
```

where these values match those in the file at https://github.com/trimesh/vhacdx/blob/main/pyproject.toml.

In other words, version constraints are needed on these as well:

```suggestion
    depends_on("py-setuptools@42:", type="build")
    depends_on("py-pybind11@2.13.1", type="build")
```